README.md

```diff
@@ -360,6 +360,58 @@ func main() {
 }
 ```
 
+### Driver Options
+
+Configuration options can be provided by the standard DSN (Data Source Name).
+
+```
+[user[:password]@]addr[/db[?param=X]]
+```
+
+#### `compress`
+
+Enable zlib compression between the client and the server.
+
+| Type      | Default   | Example                                     |
+| --------- | --------- | ------------------------------------------- |
+| boolean   | false     | user:pass@localhost/mydb?compress=true      |
+
+#### `readTimeout`
+
+I/O read timeout.
+
+| Type      | Default   | Example                                     |
+| --------- | --------- | ------------------------------------------- |
+| duration  | 0         | user:pass@localhost/mydb?readTimeout=10s    |
+
+#### `ssl`
```

Comment:
Could we rename this to `tls` or would that cause any issues?

Reply:
we can, but this is the existing `ssl` configuration so it would break existing clients. I can make it support `tls` and `ssl` options so that it's backward compatible with existing DSN's and going forward we document only the `tis` param in the readme.

Reply:
In that case, I would suggest to leave it as-is.